index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Elson Otake</title>
+</head>
+<body>
+  <header class="header">
+    <ul class="header_list">
+      <li class="company">Origami Inc</li>
+      <li class="menu">
+          <span class="material-symbols-rounded" id="hamburger">menu</span>
+      </li>
+    </ul>
+  </header>
+  <section class="presentation">
+    <div class="pres">
+      <h2 class="name">Elson Otake</h2>
```

Comment:
- [ ] Please user an `h1` tag here instead of an `h2`, as this is the main header title of the page.